Library/Homebrew/cmd/formulae.sh

```diff
@@ -0,0 +1,11 @@
+#:  * `formulae`
+#:
+#:  Prints a sorted list of locally available formulas including shortnames.
+#:
+
+# shellcheck disable=SC2155
```

Comment:
What's this for? Would be good to remove it.

Reply:
Yeah sorry looks like I needed to check everything more carefully before the commit.  Thank you catching/cleaning up a lot of it.

Reply:
`shellcheck: warning
warning - Declare and assign separately to avoid masking return values. [SC2155]`

It gets triggered any time you try to declare and assign a local variable in the same line.  It doesn't matter if one isn't using the return value of the command inside the `"$()"`.  

The alternative to disabling that check is this:

```bash
homebrew-formulae() {
  local formulae
  formulae="$(find "$HOMEBREW_REPOSITORY"/Library/Taps -type d \( -name Casks -o -name cmd -o -name .github \) -prune -false -o -name '*rb' | sed 's/\.rb//g' | sed -E 's .*/Taps/(.*)/homebrew- \1/ g'>
  local shortnames
  shortnames="$(echo "$formulae" | cut -d / -f 3)"
  echo -e "$formulae \n $shortnames" | grep -v '^homebrew/' | sort -uf
}```

which also makes the command slightly slower.  

You can see some examples with `local` on the shellcheck [wiki](https://github.com/koalaman/shellcheck/wiki/SC2155)

Reply:
> which also makes the command slightly slower, and adds two more fairly pointless lines IMO.

Are you sure it's slower? How much slower?

Yeh, I think the separate declarations is preferable 👍🏻

Reply:
Well, I can't say how or why, but there was a definite speed impact how it was originally written... but the newer version with several changes based on everyone's suggestions seems to have fixed that speed hit.  Or maybe it was just a fluke?  I don't really know :(.  Regardless, I've made the suggested changes.